### PR TITLE
Update set-proxy-settings.md

### PR DESCRIPTION
**Pull request by abhilashaagarwala**

### CODE REVIEW

**Comment by PRMerger14** [comment]
@abhilashaagarwala : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by mattbriggs — APPROVED** [review]

**Comment by mattbriggs** [comment]
#sign-off

**Review by nwoodmsft — COMMENTED** [review]
Are the links in the "## (Optional) Install PowerShell Modules using a Proxy Server" section working? It might just be the github/pr preview, but clicking those links gave me 404.

**Comment by abhilashaagarwala** [comment]
yep they are working, links in PR are not clickable

File: azure-stack/aks-hci/set-proxy-settings.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ When you run this command, the following are excluded:
 - Internal Kubernetes service traffic (.svc) where _.svc_ represents a wildcard name. This is similar to saying *.svc, but none is used in this schema.
 - The private network address space (10.0.0.0/8,172.16.0.0/12,192.168.0.0/16). Note that the private network address space contains important networks, such as the Kubernetes Service CIDR (10.96.0.0/12) and Kubernetes POD CIDR (10.244.0.0/16).
 
-While these default values will work for many networks, you may need to add more subnet ranges and/or names to the exemption list. For example, you may want to exempt your enterprise namespace (.contoso.com) from being directed through the proxy. You can achieve that by specifying the values in the proxyServer NoProxy list:
+While these default values will work for many networks, you may need to add more subnet ranges and/or names to the exemption list. For example, you may want to exempt your enterprise namespace (.contoso.com) from being directed through the proxy. You can achieve that by specifying the values in the proxyServerNoProxy list:
```
Comment (nwoodmsft) [inline azure-stack/aks-hci/set-proxy-settings.md:73]: Looks like this got changed accidentally - we need the space between as the parameter name is "noProxy".

Can we say:

"You can achieve that by specifying the values in the -noProxy list:"

File: azure-stack/aks-hci/set-proxy-settings.md [hunk 2]
```diff
@@ -79,6 +79,38 @@ Set-AksHciConfig -proxySetting $proxySetting -...
...
+```
+Configure machine-wide proxy exclusions on **each** of the physical cluster hosts where the problem was detected.
+
+> [!NOTE]
+> We recommend having the same proxy settings on all the nodes in the failover cluster. Having different proxy settings on different physical nodes in the failover cluster may lead to unexpected issues.
+
+Run the following PowerShell script and replace the $no_proxy parameter string with a suitable NO_PROXY exclusion string for your environment.
+
+```powershell
+$no_proxy = localhost,127.0.0.1,.svc,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16"
```
Comment (nwoodmsft) [inline azure-stack/aks-hci/set-proxy-settings.md:107]: By bad, I missed a quote on this string...also lets just reemphasize here that it's likely they will need to exempt their domain name (the prior section on exclusions mentions that).

Can we update this line to have the missing quote and the domain suffix example:

$no_proxy = "localhost,127.0.0.1,.svc,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16,.contoso.com"

**Comment by sethmanheim** [comment]
@abhilashaagarwala @nwoodmsft Just FYI for future reference, if you submit the PR in the private repo instead, you will get build validation checks and preview links (with links that work), which can help verify the content.

File: azure-stack/aks-hci/set-proxy-settings.md [hunk 3]
```diff
@@ -79,6 +79,38 @@ Set-AksHciConfig -proxySetting $proxySetting -...
...
+        Write-Host "Problem Detected! A machine-wide proxy server is configured, but no proxy exclusions are configured"
+    }
+}
+```
+Configure machine-wide proxy exclusions on **each** of the physical cluster hosts where the problem was detected.
+
+> [!NOTE]
+> We recommend having the same proxy settings on all the nodes in the failover cluster. Having different proxy settings on different physical nodes in the failover cluster may lead to unexpected issues.
+
+Run the following PowerShell script and replace the $no_proxy parameter string with a suitable NO_PROXY exclusion string for your environment.
```
Comment (nwoodmsft) [inline azure-stack/aks-hci/set-proxy-settings.md:104]: Do you think here we should give a hint to look back at the prior section on exclusions for more information on crafting the right NO_PROXY setting for their environment?

Comment (abhilashaagarwala) [reply azure-stack/aks-hci/set-proxy-settings.md:104]: ack

**Comment by PRMerger4** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by mattbriggs — APPROVED** [review]

**Comment by mattbriggs** [comment]
#sign-off

**Pull request merged**